src/pages/pages/login/index.js

```diff
@@ -69,7 +70,10 @@ const LoginPage = () => {
   // ** Hook
   const theme = useTheme()
   const router = useRouter()
-
+  const { status, data } = useSession();
+  useEffect(() => {
```

Comment:
Also this useEffect it will be better to do authentication verification before even loading the page that contains the Login Component we can use it by getServerSideProps